docs/configuration.md

```diff
@@ -5,9 +5,10 @@ title: Configuration
 
 ## Environment Variables
 
-The following are environment variables used by the app:
+Honeycomb relies on environment variables during build and run time to control the different configurations. Bellow we list the  environment variables used by the app and indicate whether they are mandatory, optional and needed during build (`npm build`) or run time (using the application executable of `npm run dev`)
 
-* `ELECTRON_START_URL` [string]: URL (e.g. `http://localhost:3000`) where the front end of the app is being hosted - also used in `electron.js` to indicate the app is running in dev mode
-* `EVENT_MARKER_PRODUCT_ID` [string]: The product ID of the event marker (e.g. `0487`).  If not set, it will use the `productID` set in `public/config/trigger.js`.
-* `REACT_APP_AT_HOME` [boolean]: whether the app is being used in home mode (true) or clinic mode (false)
-* `REACT_APP_PATIENT_ID` [string]: The default patient id to show when requesting a patient ID in `userID`.  If not set, no default is shown (blank input box).
+* `ELECTRON_START_URL` ![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow): URL (e.g. `http://localhost:3000`) where the front end of the app is being hosted - also used in `electron.js` to indicate the app is running in dev mode
+* `EVENT_MARKER_PRODUCT_ID`![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow)![](https://img.shields.io/badge/-runtime-purple): The product ID of the event marker (e.g. `0487`).  If not set, it will use the `productID` set in `public/config/trigger.js` if available, or attempt to connect using the com name.
+* `EVENT_MARKER_COM_NAME` ![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow)![](https://img.shields.io/badge/-runtime-purple): The com name of the event marker (e.g. `COM3`). If not set, it will use the `comName` set in `public/config/trigger.js`.  If the productID is set (not an empty string), this field will be ignored.
+* `REACT_APP_AT_HOME`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-runtime-purple)![](https://img.shields.io/badge/-builtime-blue): whether the app is being used in home mode (true) or clinic mode (false). During development and build time, you can run `npm run dev-home`/`npm run dev-clinic` or `npm run build-home`/`npm run build-clinic` to have the npm script set the variable for you. Wen running the production desktop application, you need to define it.
```

Comment:
`builtime-blue` => `buildtime-blue`?

Reply:
`dev-home` & `dev-clinic` => `dev:home` & `dev:clinic`, same with `build`

Reply:
`Wen` => `When`

Reply:
Could probably be more clear that not setting it is the same as setting it to false.  And that the value at buildtime needs to match the value at runtime.